modules/tensor_mechanics/src/materials/RecomputeRadialReturn.C

```diff
@@ -116,3 +118,12 @@ RecomputeRadialReturn::computeInelasticStrainIncrement()
 
   computeStressFinalize(_inelastic_strain_increment[_qp]);
 }
+
+Real
+RecomputeRadialReturn::getIsotropicShearModulus()
+{
+  Real shear_modulus = _elasticity_tensor[_qp](1,2,1,2);
+  if ((_mesh.dimension() == 3) && (shear_modulus != _elasticity_tensor[_qp](1,3,1,3)))
```

Comment:
True.  This was code moved from RecomputeRadialReturnIsotropicPlasticity.  Want me to remove them?